app/resources/base.html.template

```diff
@@ -16,101 +16,92 @@
-  <meta name="keywords" content="{{config.keywords}}">
-  <meta http-equiv="content-type" content="text/html; charset={{env.charset}}">
-
-  {% if not env.amp %}
-  <meta name="viewport" content="width=device-width, user-scalable=yes, initial-scale=1" />
-  {% else %}
-  <meta name="viewport" content="width=device-width,minimum-scale=1">
-  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-  <script async src="https://cdn.ampproject.org/v0.js"></script>
-  {% endif %}
+    <meta charset="utf-8">
+    {% include "gtm.html.template" %}
+    <meta name="keywords" content="{{config.keywords}}">
+    <meta http-equiv="content-type" content="text/html; charset={{env.charset}}">
+
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    {% if env.amp %}
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    {% endif %}
 
-  {% if not env.robots_ok %}<meta name="robots" content="noindex">{% endif %}
-  {% if feed_url %}
-    <link rel="alternate" type="application/atom+xml"
+    {% if not env.robots_ok %}<meta name="robots" content="noindex">{% endif %}
+    {% if feed_url %}
+      <link rel="alternate" type="application/atom+xml"
         title="{% trans 'Atom feed of updates about this person' %}"
         href="{{feed_url}}">
-  {% endif %}
-  {% if not env.amp %}
-  <link rel="alternate" media="handheld" href="{{env.url}}">
+    {% endif %}
+    {% if env.amp %}
+    <link rel="canonical" href="{{env.canonical_url}}">
```

Comment:
Inde this line and line 44 by one more level.

Reply:
done.